tests/test_workers/test_scm.py

```diff
@@ -89,6 +94,9 @@ def test_clone_and_archive(
         mock_ugs.assert_called_once_with(mock_clone.return_value)
     else:
         mock_ugs.assert_not_called()
+    #
```

Comment:
should it be empty?

Reply:
It should not. Holiday mood was impeding my brain. :D